src/main/java/io/dropwizard/flyway/FlywayBundle.java

```diff
@@ -26,4 +26,8 @@ public FlywayFactory getFlywayFactory(T configuration) {
         // Default Flyway configuration
         return new FlywayFactory();
     }
+
+    public String name() {
```

Comment:
This method could be `protected`.

Reply:
I updated jtunison:master to use `protected String name()`, but it looks like the PR isn't updating.  (GitHub bug?) Want me to close this PR & create a second one?

Reply:
@jtunison I think that was GitHub being a bit slow today.